socs/agents/stm_encoder/agent.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+'''OCS agent for stimulator encoder
+'''
+import argparse
+import os
+import time
+
+import txaio
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import TimeoutLock
+
+from socs.agents.stm_encoder.drivers import (PATH_LOCK, StmEncReader,
+                                             get_path_dev)
+
+
+class StmEncAgent:
+    '''OCS agent class for stimulator encoder
+    '''
+
+    def __init__(self, agent, path_dev=None, path_lock=PATH_LOCK):
+        '''
+        Parameters
+        ----------
+        path_dev : str or pathlib.Path
+            Path to the generic-uio device file for str_rd IP.
+        path_lock : str or pathlib.Path
+            Path to the lockfile.
+        '''
+        self.active = True
+        self.agent = agent
+        self.log = agent.log
+        self.lock = TimeoutLock()
+        self.take_data = False
+
+        if path_dev is None:
+            self._path_dev = get_path_dev()
+
+        self._dev = StmEncReader(self._path_dev, path_lock, verbose=False)
+
+        self.initialized = False
+
+        agg_params = {'frame_length': 60}
+        self.agent.register_feed('stm_enc',
+                                 record=True,
+                                 agg_params=agg_params,
+                                 buffer_time=1)
+
+    def acq(self, session, params):
+        '''Starts acquiring data.
+        '''
```

Comment:
Follow the process docstring formatting recommendations here:
```suggestion
        '''acq()
        
        **Process** - Start acquiring data.
        '''
```
Also add `session.data` structure to the docstring.